.github/workflows/deploy-demo-server.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy Demo Server
+
+on:
+  # Manual trigger
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: Deploy
+    environment: Production
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          cache: 'npm'
+
+      - run: npm install
+
+      - name: Deploy
+        uses: cloudflare/wrangler-action@v3
+        with:
+          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+          workingDirectory: "./example/demo"
```

Comment:
Hard to test this in a PR, but should work. If it doesn't, I will send a fix directly to master in a follow-up.